src/main/kotlin/app/revanced/patcher/arsc/Resource.kt

```diff
@@ -0,0 +1,83 @@
+package app.revanced.patcher.arsc
```

Comment:
I made the resource classes/functions because the patches should ideally not have to directly interact with aapt, arsclib or whatever is being used.

The name `resources` probably makes more sense than `arsc` though (the other things in that package will be moved out).

Reply:
ReVanced patcher makes use of Arsclib. Arsclib is for resources, so the logic for resources should be in Arsclib. You can create proxy methods for all the Arsclib methods if you want to have that layer of abstraction for any reason, but the logic for resources should be in Arsclib. ReVanced Patcher is merely an orchestration of dexlib, arsclib & provides its own toolset, such as patches or fingerprints.

Reply:
Right, that is a valid point. I would still argue that we should wrap ARSCLib because it does not have documentation unlike aapt, androlib or dexlib2.

I am not entirely convinced it makes sense to move the classes and functions in `Resource.kt` to ARSCLib, because they are just wrappers that don't really implement anything that does not already exist in ARSCLib.